assets/js/components/Dashboard/PaessDetailContainer.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import PaessScreenDetail from "./PaessScreenDetail";
+import { Screen } from "../../models/screen";
+import { Row, Col } from "react-bootstrap";
+
+interface PaessDetailContainerProps {
+  screens: Screen[];
+}
+
+const PaessDetailContainer = (
+  props: PaessDetailContainerProps
+): JSX.Element => {
+  const zonePositions = {
+    left: ["s", "w"],
+    center: ["c", "m"],
+    right: ["n", "e"],
+  };
+
+  const stationCode = props.screens[0].station_code;
+  const leftScreens = props.screens.filter(
+    (screen) => screen.zone != null && zonePositions.left.includes(screen.zone)
+  );
+  const centerScreens = props.screens.filter(
+    (screen) =>
+      screen.zone != null && zonePositions.center.includes(screen.zone)
+  );
+  const rightScreens = props.screens.filter(
+    (screen) => screen.zone != null && zonePositions.right.includes(screen.zone)
+  );
+
+  return (
+    <Row className="h-100" md={3}>
+      <Col
+        key={`${stationCode}-left`}
+        className={`screen-detail__iframe-container screen-detail__iframe-container--pa_ess`}
+        data-testid="paess-col-left"
+      >
+        {leftScreens.length > 0 && (
+          <div>
```

Comment:
Should be able to remove this `div`.

Reply:
So I mainly added this `div` to achieve the center alignment that I was looking for in the cases where there are two signs in a given column. With one sign, the center alignment looks just fine, but when there's two they end up side by side in the column instead of vertically stacked. Adding the div seemed to do the trick, but certainly let me know if there's a better way to go about it!